File: Questoes/1368A-C+.py
t = int(input())

for i in range(t):
	a, b, c = [int(x) for x in input().split()]
	cont = 0
	
	while (a <= c and b <= c):
		if a <= b:
			a += b
		elif b < a:
			b += a
		cont += 1
	print(cont)

File: Questoes/1368A-C+.cpp
#include <iostream>

using namespace std;

int main(){
	
	int t;
	cin >> t;
	
	while (t--){
		
		int a, b, n;
		cin >> a >> b >> n;
		int cont = 0;
		
		while (a <= n && b <= n){
			if (a <= b)
				a += b;
			else
				b += a;
			cont += 1;
		}
		
		cout << cont << endl;
		
	}

	return 0;
}
